.github/workflows/testNginxForAzureDeployV1.yml

```diff
@@ -0,0 +1,55 @@
+# File: .github/workflows/testNginxForAzureDeployV1.yml
+
+name: Test Github action to update NGINX for Azure configurations and certificates
+on:
+  - push
+  - workflow_dispatch
+
+env:
+  NGINX_DEPLOYMENT_NAME: '0707221656'
+  NGINX_TRANSFORMED_CONFIG_DIR_PATH: /etc/nginx/
+  NGINX_ROOT_CONFIG_FILE: nginx.conf
+  TEST_RESOURCE_GROUP_NAME: george-test
+  NGINX_RESOURCE_LOCATION: westcentralus
+  NGINX_CERT_DETAILS: '[{"nginx_cert_name": "server4", "keyvault_secret": "https://isolated-kv.vault.azure.net/secrets/server", "certificate_virtual_path": "/etc/ssl/certs/server4.crt", "key_virtual_path": "/etc/ssl/certs/server4.key"  }, {"nginx_cert_name": "server5", "keyvault_secret": "https://isolated-kv.vault.azure.net/secrets/acr-auth", "certificate_virtual_path": "/etc/ssl/certs/server5.crt", "key_virtual_path": "/etc/ssl/certs/server5.key"  }]'
```

Comment:
Is it possible to break this string over multiple lines instead of having one really long horizontal line?

Reply:
Why are these keys `snake_case` instead of `camelCase` which is established code style of the APIs?

Reply:
> Why are these keys `snake_case` instead of `camelCase` which is established code style of the APIs?

Changed this to camelCase. Thanks

Reply:
> Is it possible to break this string over multiple lines instead of having one really long horizontal line?

Unfortunately, no. I explored the possibility of providing this input in some yml array style but there's no support for it. The only possible values for environment variables are strings, integers and booleans